overlore/jsonrpc/methods/spawn_npc/constants.py

```diff
@@ -1,3 +1,14 @@
-NPC_PROFILE_SYSTEM_STRING = """Imagine you're the game master for a fantasy strategy game, you are tasked with creating a Non-Playable Character."""
+NPC_PROFILE_SYSTEM_STRING = """Imagine you're the game master for a fantasy strategy game, you are tasked with creating the full name, backstory and a character trait for a Non-Playable Character.
+Output format example:
+{
```

Comment:
This is not necessary to add and can be removed. The guard already adds a JSON schema of the `BaseModel` class to the instructions prompt

Reply:
I'll do some more testing to be sure, but it seemed that the the prompt wasn't getting through the validator without it

Reply:
It seems to work fine without the explicit format mention indeed. I stand corrected.